src/motile_plugin/backend/solve.py

```diff
@@ -51,13 +52,14 @@ def solve(
     """
     if input_data.ndim == 2:
         cand_graph = get_candidate_graph_from_points_list(
-            input_data, solver_params.max_edge_distance
+            input_data, solver_params.max_edge_distance, scale=scale
```

Comment:
I didn't add scale to this function in the toolbox yet. We do need it. But, our output points will be in world units, and we should treat them accordingly when visualizing.